src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1832,6 +1884,8 @@ private void WorkspaceAdded(WorkspaceModel item)
                 }
                 workspaces.Add(newVm);
             }
+
+            CheckOnlineAccess();
```

Comment:
Why does the check need to be called here? How can adding a workspace change the online status?

Reply:
I think more, checking something in the "background", just in case the status changed between last check. People are less likely to notice something going on, if they are already waiting.